apps/src/templates/teacherDashboard/EditSectionForm.story.jsx

```diff
@@ -1,74 +1,85 @@
 import React from 'react';
 import {UnconnectedEditSectionForm as EditSectionForm} from './EditSectionForm';
 import {action} from '@storybook/addon-actions';
-import {SectionLoginType} from '@cdo/apps/util/sharedConstants';
 import {testSection, courseOfferings} from './teacherDashboardTestHelpers';
 
-export default storybook => {
-  storybook = storybook.storiesOf('EditSectionForm', module);
+export default {
+  title: 'EditSectionForm',
+  component: EditSectionForm
+};
+
+const noStudentsSection = {
+  ...testSection,
+  studentCount: 0,
+  participantType: 'student'
```

Comment:
Since all participantTypes are 'student', can we pull that into the 'testSection'?

Reply:
What a fun cleanup! This was from the test helper file, so I checked the other places this was used and they're all participantType = 'student' as well! And then I noticed the other test section also has a participantType student set everywhere, so I pulled both into the helper test sections and then updated everything they touched.